automated/linux/network-manager/network-manager.yaml

```diff
@@ -0,0 +1,27 @@
+metadata:
+    name: network-manager-smoke
+    format: "Lava-Test Test Definition 1.0"
+    description: "The test runs basic nmcli subcommands"
+    maintainer:
+        - milosz.wasilewski@foundries.io
+    os:
+        - debian
+        - ubuntu
+        - fedora
+        - centos
+    scope:
+        - functional
+    environment:
+        - lava-test-shell
+    devices:
+        - hi6220-hikey
+        - apq8016-sbc
+        - imx8mmevk
+        - imx6ullevk
+params:
+    INTERFACE: "eth0"
```

Comment:
does it make sense to add skip_install ?

Reply:
added. Sorry for delay. I took a few days off.